qcut/docs/completed/bug_v3.md

```
@@ -0,0 +1,357 @@
+(anonymous) @ index-Bp5JSGTs.js:425
+Js @ index-Bp5JSGTs.js:425
+(anonymous) @ index-Bp5JSGTs.js:428
+Tr @ vendor-CvAI8bIM.js:138
+D0 @ vendor-CvAI8bIM.js:151
+mp @ vendor-CvAI8bIM.js:151
+ab @ vendor-CvAI8bIM.js:151
+ab @ vendor-CvAI8bIM.js:151
+ab @ vendor-CvAI8bIM.js:151
+ab @ vendor-CvAI8bIM.js:151
+N0 @ vendor-CvAI8bIM.js:151
+C_ @ vendor-CvAI8bIM.js:151
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)